packages/coil-extension/src/popup/components/views/SettingsView.tsx

```diff
@@ -95,18 +95,21 @@ export const SettingsView = () => {
     }
 
     if (user?.email) {
-      return user.email
+      const emailHandle = user.email.split('@')[0]
+      return emailHandle
     }
   }
 
   const getSecondaryName = () => {
     if (user?.fullName && user?.shortName) {
       return user.shortName
     } else {
-      if (user?.shortName) {
+      if (user?.shortName || user?.fullName) {
         return user.email
+      } else {
+        const emailDomain = user.email.split('@')[1]
+        return `@${emailDomain}`
```

Comment:
So, secondary name can be "gmail.com"?  lol!

Reply:
Yup, in this instance is will be MyEmailAddress for the top line and @Domain.com on the second line. I'm splitting the email in the event that there is no shortName and no fullName. This will help a little with really long emails while keeping the same aesthetic.